webapp/Connector/src/view/PageHeader.js

```diff
@@ -145,6 +147,42 @@ Ext.define('Connector.view.PageHeader', {
             }, this)
         });
 
+        if (this.model.data.assay_identifier) {
```

Comment:
Can this be switched to use configuration information for the components in `cube.js`? I was thinking about something similar to the `hasSearch` property for the details tabs in `cube.js:565`

Reply:
Yup, good suggestion, now switched to use config info similar to hasSearch property